ansible/roles/test/files/ptftests/fib.py

```diff
@@ -44,12 +44,15 @@ def get_next_hop_list(self):
     # Initialize FIB with FIB file
     def __init__(self, file_path):
         self._ipv4_lpm_dict = LpmDict()
-        for ip in EXCLUDE_IPV4_PREFIXES:
-            self._ipv4_lpm_dict[ip] = self.NextHop()
+        # no need to add EXCLUDE_IPV4_PREFIXES into test
```

Comment:
could you point out why about this code change?

Reply:
As the name of this variable, "EXCLUDE_IPV4_PREFIXES" should be a list of IP ranges that we need to exclude from decap test. Somehow this script join those excluded IP ranges into test at this section and cause fail the decap test while checking incorrect IP ranges. The simple way to prevent this issue is just comment out this section.